lib/rust/ensogl/core/src/display/world/stats.rs

```diff
@@ -41,30 +46,47 @@ impl {
             monitor.add( debug::monitor::SpriteSystemCount  :: new(&stats) ),
             monitor.add( debug::monitor::SpriteCount        :: new(&stats) ),
         ];
-        Self {stats,monitor,panels}
+        Self {stats,performance,forced_profiling,monitor,panels}
+    }
+
+    /// Force data measurements to be always performed, regardless of whether the monitor is
+    /// visible.
+    pub fn force_profiling(&mut self) {
```

Comment:
Please fix this and other names in this PR. Is `MonitorData.force_profiling` understandable to a reader? If you'd approach anyone in the Rust team and ask what this function do, would they understand that this function has ... nothing to do with profiling and it just enables stats gathering? Please take care about naming! Otherwise, the PRs will be in the review process for a long time.